ui/js/models/resultsets_store.js

```diff
@@ -143,11 +144,10 @@ treeherder.factory('ThResultSetStore', [
                     _.pick($location.search(), ["exclusion_profile", "visibility"]));
             }
             lastPolltime = Date.now();
-
             jobUpdatesPromise.then(function(jobList) {
+                jobList = _.flatten(jobList);
                 if (jobList.length > 0) {
                     lastJobUpdate = getLastModifiedJobTime(jobList);
```

Comment:
Again, wouldn't this just have the effect of masking errors? If we don't have a last_modified field on a job object, it's an indication that something is very wrong, either on the client or the server. In that case, isn't it desirable to show a popup that gets the user to reload the page?